pyproject.toml

```diff
@@ -43,6 +43,7 @@ redis = {version = "^4.3.4", optional = true}
 django-oauth-toolkit = "^2.3.0"
 mozilla-django-oidc = "^3.0.0"
 urllib3 = "^1.26.0,<2.0.0"  # pinned because of poetry issues with urllib3 2.0.0
+pyyaml = "^6.0.1"
```

Comment:
why is this required? is this just a missing dependency in another package?

Reply:
It's required for the SchemaView of drf. See the documentation there...

Reply:
![image](https://github.com/ephios-dev/ephios/assets/5970076/e741a94a-1f13-4697-82ea-142a31f597f4)
https://www.django-rest-framework.org/api-guide/schemas/

Reply:
wow. still think that it _should_ be their job to specify that ^^

Reply:
well its really an optional feature but yeah